src/Workspace/Wes_Timing_System_4/TimingScript/TimingLeaderstatsService.lua

```diff
@@ -57,7 +59,7 @@ end
 --- Adds leaderstats to a new player
 ---@param player Player The new player
 function service.AddPlayer(player: Player)
-    
+    require(script.Parent.TimingCommandsService).AddCommandsToChat(player)
```

Comment:
Use `require()` at the beginning of the file.

Reply:
Perhaps this comment wasn't visible because I added it as conversation. Please address this

Reply:
This is the only solution possible as placing the `require()` at the beginning of the file causes an infinite `require()` loop